libcontainer/user/user.go

```diff
@@ -433,9 +431,11 @@ func GetAdditionalGroups(additionalGroups []string, group io.Reader) ([]int, err
 // that opens the groupPath given and gives it as an argument to
 // GetAdditionalGroups.
 func GetAdditionalGroupsPath(additionalGroups []string, groupPath string) ([]int, error) {
-	group, err := os.Open(groupPath)
-	if err == nil {
```

Comment:
The intention of this code was so that you could pass `""` as a path and you won't get any errors unless the specification of a user requires lookup -- effectively allowing the numerical specification of a user without specifying `/etc/passwd`.

Obviously I got that a bit wrong, but the current version your proposing doesn't retain that behaviour. Though maybe we should consider whether it makes sense for us to keep that behaviour.